src/pages/index.js

```diff
@@ -58,7 +58,7 @@ function Home() {
   return (
     <Layout
       title={siteConfig.title}
-      description="Description will go into a meta tag in <head />">
+      description="Spacemacs like keybindings for Visual Studio Code">
```

Comment:
Can you move the string to `docusaurus.config.js` as a `description` like `tagline`?

Reply:
```suggestion
      description={siteConfig.tagline}>
```